src/Twig/RichEditorExtension.php

```diff
@@ -90,6 +87,25 @@ public function renderRichEditorField(string $content)
         return $this->renderElements($elements);
     }
 
+    /**
+     * @param string $content
+     *
+     * @throws LoaderError
+     * @throws RuntimeError
+     * @throws SyntaxError
+     *
+     * @return array
+     */
+    public function getRichEditorFieldElements(string $content): array
+    {
+        $elements = json_decode($content, true);
+        if (!\is_array($elements)) {
+            return [$elements];
```

Comment:
I'm not a huge fan of this.
In that case you could return `['foo']` (array of text elements, one actually) instead of an array of Ui Elements.